words/objects.txt

```diff
@@ -1872,14 +2414,18 @@ shaker
 shallot
 shame
 shampoo
+shamrock
 shape
 share
 shark
+shasta
```

Comment:
hmm I would probs remove https://en.wikipedia.org/wiki/Shasta

Reply:
because of the tribe correlation?

Reply:
yeah definitely, also because it's a brand name and a deity name

Reply:
I've only ever heard it used as a flower, TIL!